src/app/components/rpc-transaction-request/post-conditions/post-condition-list.tsx

```diff
@@ -0,0 +1,46 @@
+import { type PayloadWire, PostConditionType, type PostConditionWire } from '@stacks/transactions';
+
+import { FungiblePostConditionItem } from './fungible-post-condition-item';
+import { PostConditionItem } from './post-condition-item';
+
+interface PostConditionListProps {
+  payload: PayloadWire;
+  postConditions: PostConditionWire[];
+}
+// TODO: Refactor post conditions
+//
+// This code has, for the most part, simply been relocated and isolated from existing code
+// that handles post conditions for legacy tx requests. The intention here is to first isolate
+// the code, then refactor it for Approver UX with new tests.
+export function PostConditionList({ payload, postConditions }: PostConditionListProps) {
+  return (
+    <>
+      {postConditions.map((pc, index) => {
+        if (pc.conditionType === PostConditionType.Fungible) {
```

Comment:
```suggestion
export function PostConditionList({ payload, postConditions }: PostConditionListProps) {
  return postConditions.map((pc, index) => {
    if (pc.conditionType === PostConditionType.Fungible) {
```

in newer ts we can return directly